src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue13537.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue13537 : _IssuesUITest
+		{
+#if ANDROID
+		const string backButtonIdentifier = "";
+#else
+        const string backButtonIdentifier = "Home";
+#endif
+		public Issue13537(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+			public override string Issue => "ApplyQueryAttributes should Trigger for all navigations";
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ApplyQueryAttributeShouldTriggerforPushAndPopButton()
+		{
+			App.WaitForElement("HomePageTestLabel");
+			App.Tap("PushAsyncButton");
+			App.WaitForElement("InnerPageTestLabel");
+			App.Tap("PopAsyncButton");
+			var result = App.WaitForElement("HomePageTestLabel").GetText();
+			Assert.That(result , Is.EqualTo("Issue13537HomePage QueryAttribute is triggered"));
+		}
+
+#if !MACCATALYST
```

Comment:
Why we avoid this test on Catalyst?

Reply:
Hi @jsuarezruiz  , `App.TapBackArrow` does not actually tap the back arrow button. Instead, it taps the first tab element on the screen because both elements share the same accessibility identifier. To address this, Catalyst has been ignored, and appropriate comments have been added.